src/helpers/tokens/generate.token.js

```diff
@@ -2,11 +2,9 @@ import jwt from 'jsonwebtoken';
 
 const createToken = (user, secret) => {
   try {
-    const token = jwt.sign({ user },
-      secret,
-      {
-        expiresIn: '24h'
-      });
+    const token = jwt.sign({ user }, secret, {
```

Comment:
@MCFrank16 please remove this file from your PR

Reply:
this file is used in generating a token and it comes from @alainburindi signup and sign in PR, so if I remove it, It will cause my app to crash